packages/cashier/src/Components/eusdt-side-note.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { Modal, Text } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+
+const EUSDTSideNote = () => {
```

Comment:
Suggestion: we can combine this component to `USDSideNote` since they have identical layout and maybe pass the currency as a prop instead to render different text. This way we avoid having multiple slightly different components.

Reply:
Sounds like a plan too 👌